[2023-01-26 16:39:31,876] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Esperando_el_fichero scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2023-01-26 16:39:31,895] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Esperando_el_fichero scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2023-01-26 16:39:31,896] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 16:39:31,896] {taskinstance.py:1377} INFO - Starting attempt 6 of 8
[2023-01-26 16:39:31,896] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 16:39:31,986] {taskinstance.py:1397} INFO - Executing <Task(FileSensor): Esperando_el_fichero> on 2023-01-03 00:00:00+00:00
[2023-01-26 16:39:31,992] {standard_task_runner.py:52} INFO - Started process 280 to run task
[2023-01-26 16:39:31,996] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'Esperando_el_fichero', 'scheduled__2023-01-03T00:00:00+00:00', '--job-id', '541', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmpxaecrrzb', '--error-file', '/tmp/tmpcucf1ode']
[2023-01-26 16:39:31,998] {standard_task_runner.py:80} INFO - Job 541: Subtask Esperando_el_fichero
[2023-01-26 16:39:32,115] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.Esperando_el_fichero scheduled__2023-01-03T00:00:00+00:00 [running]> on host 1e10284898e4
[2023-01-26 16:39:32,226] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=Esperando_el_fichero
AIRFLOW_CTX_EXECUTION_DATE=2023-01-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=6
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-03T00:00:00+00:00
[2023-01-26 16:39:32,241] {base.py:68} INFO - Using connection ID 'fs_default' for task execution.
[2023-01-26 16:39:32,241] {filesystem.py:58} INFO - Poking for file /tmp/response_20230103.txt
[2023-01-26 16:39:32,241] {filesystem.py:63} INFO - Found File /tmp/response_20230103.txt last modified: 20230126163930
[2023-01-26 16:39:32,241] {base.py:301} INFO - Success criteria met. Exiting.
[2023-01-26 16:39:32,257] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=Esperando_el_fichero, execution_date=20230103T000000, start_date=20230126T163931, end_date=20230126T163932
[2023-01-26 16:39:32,288] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-26 16:39:32,356] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
